_typos.toml

```diff
@@ -7,5 +7,7 @@ extend-exclude = [
     "**/*ffi*.js",
     "**/*mock*.rs",
     "**/*data*.rs",
-    "cumulus/**"
+    "cumulus/**",
+    "smoketest/src/parachains",
+    "smoketest/src/contracts",
```

Comment:
Do we have these smoketest scripts ready? can't find in this PR.

Reply:
I experimented with writing smoketest scripts in Rust, but deleted the experiment as they were too slow to compile, and Rust's test frameworks are not as good as Golang's.

But see this in case we ever want to generate Rust clients for our pallets and contracts: https://www.notion.so/snowfork/Rust-bindings-b6566f13dd894ac1ac9a347f9bbbf160?pvs=4